packages/manager/modules/nutanix/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@ovh-ux/manager-models": "^1.4.3",
     "bootstrap4": "twbs/bootstrap#v4.0.0",
+    "ipaddr.js": "^1.9.1",
```

Comment:
Question: is this free to use? what is the licence type? is it used in other modules?

Reply:
Yes it is already used in other modules.